src/olympia/devhub/templates/devhub/includes/version_file.html

```diff
@@ -1,6 +1,6 @@
 <tr>
   <td>
-    <a href="{{ file.get_absolute_url('devhub') }}">
```

Comment:
This and below didn't fail, but caused the files to be sent as attachment instead... (because it was then passed as `attachment='devhub'`, which evaluates to `True`.)